lib/reek/context/attribute_context.rb

```diff
@@ -15,6 +15,15 @@ def full_comment
         send_expression.full_comment || ''
       end
 
+      def instance_method?
+        true
+      end
```

Comment:
I kind of see your point but I'm not sure how we would expose these values outside of the class, which is where they are used mostly.